parse_contributors_oo.py

```diff
@@ -0,0 +1,313 @@
+import json
+import ruamel.yaml
+import urllib.request
+import requests
+
+
+class processContributors:
```

Comment:
```suggestion
class ProcessContributors:
```

https://peps.python.org/pep-0008/#class-names